static/lib/main.js

```diff
@@ -2,8 +2,16 @@
 
 /* global $, window, socket, config, ajaxify, app */
 
-$('document').ready(function () {
-	$(window).on('action:ajaxify.end', function (ev, data) {
+var translations = [];
+
+require(['translator'], function(translator) {
+	translator.translate('[[qanda:topic_solved]],[[qanda:topic_unsolved]],[[qanda:thread.tool.as_question]],[[qanda:thread.alert.as_question]],[[qanda:thread.alert.make_normal]],[[qanda:thread.alert.solved]],[[qanda:thread.alert.unsolved]],[[qanda:post.alert.correct_answer"]]', function(translated) {
+		translations = translated.split(',');
+	});
+});
```

Comment:
This could be a race condition, you cannot guarantee that `translator.translate` will resolve before $(document).ready and ajaxify.end are both called. Recommend you move the `require` inside the `action:ajaxify.end` callback.